app/serializers/user_serializer.rb

```diff
@@ -22,7 +22,17 @@ def profile
     }
   end
 
-  delegate :va_profile, to: :object
```

Comment:
the entire va_profile is no longer returned by the serializer, but the User model still delegates to mvi for the va_profile (used in FormProfile)